src/registry_schemas/schemas/restoration.json

```diff
@@ -49,6 +50,7 @@
                 "approvalType": {
                     "type": "string",
                     "title": "The Type of Approval, such as Approved by Court or Registrar.",
+                    "description": "In the case of a limited restoration extension or limited restoration to full filing, the approval type needs to the same value as the original limited restoration filing.",
```

Comment:
So the approval type is required for all 4 subtypes?

Reply:
yes